kind/kind.sh

```diff
@@ -862,9 +898,11 @@ while [ $# -gt 0 ]; do
         shift
     else
         case $1 in
-            install) COMMAND="install" ;;
-            remove) COMMAND="remove" ;;
-            *) INVALID_SWITCH="${INVALID_COMMAND} $1" ; break 2 ;;
+            install-cluster) COMMAND="install-cluster" ;;
+            remove-cluster) COMMAND="remove-cluster" ;;
+            install-registry) COMMAND="install-registry" ;;
+            remove-registry) COMMAND="remove-registry" ;;
```

Comment:
Could we use instead `./kind.sh install|remove cluster|registry ?

To be discussed: If the script is finally used to manage 2 different things, should we then continue to manage everything part of the kind script or to have 2 scripts: kind.sh and registry.sh.

Reply:
Another issue is that if you do that, then you will break the existing usage that I already integrated in some of my github repo to use/install kind top of TAP, Red Hat Buildpaks, DAPR, etc

Reply:
> registry

The only way this wouldn't happen is having an independent script for the container registry.